### DISCUSSION TITLE
720p resolution option

### DISCUSSION THREAD

**Discussion by dithered** [community/community#3225] [Other Feature Feedback, Questions, & Ideas] [answered]
I saw there was a thread where you mentioned 720p wasn't reliable.... but whatever Stadia+ does works fine? You do have to refresh the browser when you switch resolutions, but it's still very useful.

It's important to be able to LOWER your bandwidth manually at times. If your connection is borderline, if you have multiple people on stadia over wifi, lots of other family traffic on your network etc. I often value consistency over resolution, especially in mutiplayer pvp games. Its the primary reason I've been using Stadia+ instead of Stadia Enhanced.

**Comment by evi-liu** [comment, marked as answer]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).
